Boop/Boop/Controllers/PopoverViewController.swift

```diff
@@ -55,18 +63,12 @@ class PopoverViewController: NSViewController {
             }
             
             if theEvent.keyCode == 36 && self.enabled { // ENTER
-                
-                guard let script = self.tableViewController.selectedScript else {
-                    return theEvent // Return event to beep
+
+                guard self.tableViewController.selectedScript != nil else {
+                    return theEvent
                 }
-                
-                
-                // Let's dismiss the popover
-                self.hide()
-                
-                // Run the script afterwards in case we need to show a status
-                self.scriptManager.runScript(script, into: self.editorView)
-                
+
+                self.runScriptAgain()
```

Comment:
This should be

```suggestion
                self.runSelectedScript()
```

Reply:
Oh boy! Xcode's autocomplete betrayed me here. Nice catch, fixed 👍